PlayCover/Views/AppSettingsView.swift

```diff
@@ -156,6 +156,11 @@ struct KeymappingView: View {
                     Toggle("settings.toggle.autoKM", isOn: $settings.settings.noKMOnInput)
                         .help("settings.toggle.autoKM.help")
                 }
+                HStack {
+                    Toggle("settings.toggle.enableScrollWheel", isOn: $settings.settings.keymapping)
+                        .help("settings.toggle.enableScrollWheel.help")
```

Comment:
For i18n to work, you need to provide an English translation of these strings. Just edit the English localizable strings file. Translations for other languages will be automatically handled via Weblate.